zio-http/src/main/scala/zio/http/Body.scala

```diff
@@ -170,19 +209,27 @@ object Body {
   /**
    * Helper to create Body from Stream of bytes
    */
-  def fromStream(stream: ZStream[Any, Throwable, Byte]): Body = {
-    new Body {
-      override def isComplete: Boolean = false
+  def fromStream(stream: ZStream[Any, Throwable, Byte]): Body = new StreamBody(stream)
 
-      override def asChunk(implicit trace: Trace): Task[Chunk[Byte]] = stream.runCollect
+  private[zio] final case class StreamBody(stream: ZStream[Any, Throwable, Byte]) extends AnyVal with Body {
```

Comment:
Extending AnyVal can easily slow things down due to wrapping / unwrapping that occurs in any generic context.

Reply:
Will remove it.